lib/app_store_connect.rb

```diff
@@ -5,6 +5,7 @@
 require 'app_store_connect/object/id'
 require 'app_store_connect/object/attributes'
 require 'app_store_connect/object/properties'
+require 'app_store_connect/object/included'
```

Comment:
you might want to consider using https://github.com/fxn/zeitwerk for requiring

Reply:
Added in #145

Reply:
sweet!